sql-plan-management.md

```diff
@@ -120,6 +120,69 @@ SELECT * FROM test . t WHERE a > ?
 > ```
 >
 > 因此包含单个常量的 SQL 语句和包含被逗号连接起来多个常量的 SQL 语句，在被绑定时会被 TiDB 视作不同的 SQL 语句，需要分别创建绑定。
+> 例如：
+> 
+> ```sql
+> create table t(a int, b int, key idx(a));
+> create session binding for select * from t where a in (?) using select /*+ use_index(t, idx) */ * from t where a in (?);
+> show bindings;
+> +-----------------------------------------------+----------------------------------------------------------------------+------------+---------+-------------------------+-------------------------+---------+--------------------+--------+------------------------------------------------------------------+-------------+
+> | Original_sql                                  | Bind_sql                                                             | Default_db | Status  | Create_time             | Update_time             | Charset | Collation          | Source | Sql_digest                                                       | Plan_digest |
```

Comment:
Sometimes our result outputs are too wide, making it inconvenient to view. Can we use "\G" to improve the output?

Reply:
@hawkingrei The current preview of this code block is:

<img width="606" alt="image" src="https://github.com/pingcap/docs-cn/assets/60599231/6a78679c-2cfe-4bc8-838a-daa83263f03f">


To improve the readability and user experience of wide code blocks in our documentation site, our FE team plans to support "toggle word wrap" in the future. For example:
![7b84b669-12a5-4add-9a6e-92ff340254d4](https://github.com/pingcap/docs-cn/assets/60599231/93d83b1b-38ec-49ac-bb91-132c20b33f28)
![dcb47e04-a249-4746-a3e9-94d9aa9b5e3d](https://github.com/pingcap/docs-cn/assets/60599231/f6316883-2812-49f2-9008-8ca7853dca2b)